css/syntaxes.json

```diff
@@ -98,6 +110,12 @@
   "compositing-operator": {
     "syntax": "add | subtract | intersect | exclude"
   },
+  "compound-selector": {
+    "syntax": "[ <type-selector>? <subclass-selector>*[ <pseudo-element-selector> <pseudo-class-selector>* ]* ]!"
```

Comment:
There should be a space here, so not:
```
<subclass-selector>*[
```
...but
```
<subclass-selector>* [
```

I don't know if whitespace changes the meaning, but we might as well fix this.